spec/recipes/omnibus_toolchain_spec.rb

```diff
@@ -1,11 +1,17 @@
 require 'spec_helper'
 
 describe 'omnibus::_omnibus_toolchain' do
-  let(:chef_run) { ChefSpec::SoloRunner.converge(described_recipe) }
+  let(:chef_run) {
+    ChefSpec::SoloRunner.new do|node|
+      node.automatic['kernel']['machine'] = 'x86_64'
+      node.automatic['kernel']['bits'] = '64'
+      node.automatic['architecture'] = 'x86_64'
```

Comment:
Removed. The tests still pass, so it looks like the mocking is setup to default to 64 bit. The other possibility is that it just takes the machine type of where the tests are ran - meaning it would default to 32 bit on a 32 bit system. I am fine with leaving these mocks out until we encounter that scenario though.